Panorama add description field to commit command (#28963)
demisto/content: Packs/PAN-OS/ReleaseNotes/2_1_2.md

#### Integrations

##### Palo Alto Networks PAN-OS

> - Fixed a problem where the ***pan-os-commit*** command did not take into account the *description* argument.

Reviewer: ```suggestion
- Fixed an issue where the ***pan-os-commit*** command did not take into account the *description* argument.
```
